client/cmd/dexc/main.go

```diff
@@ -64,7 +64,7 @@ func main() {
 	})
 	if err != nil {
 		fmt.Fprint(os.Stderr, "error creating client core: ", err)
-		return
+		os.Exit(1)
```

Comment:
So I think this is all OK since there are no defers in `main`, but we'll need a main wrapper in the future most likely.  I say that partly because the clean way to shut this down in event of an error would be to `cancel(); return` with a `defer wg.Wait()` so that even if one subsystem fails "(or the rpc and web options conflict), any ones that are already running can shutdown cleanly.